keycloak-iam/src/main/java/com/valtimo/keycloak/liquibase/changelog/ChangeLog20240116MigrateTaskAssigneeEmailToUserId.kt

```diff
@@ -57,14 +57,16 @@ class ChangeLog20240116MigrateTaskAssigneeEmailToUserId : CustomTaskChange, Envi
                 if (!EmailValidationUtil.isValidEmail(taskAssigneeEmail)) {
                     logger.error { "Failed to migrate task assignee. Invalid email: '$taskAssigneeEmail' for task '$taskId'" }
                 } else {
-                    var taskAssigneeUserId: String? = null
                     try {
-                        taskAssigneeUserId = getKeycloakUserIdByEmail(taskAssigneeEmail)
-                    } catch (_: Exception) {
+                        val assigneeValue = getKeycloakUserIdByEmail(taskAssigneeEmail)
+                        updateTaskInDatabase(connection, taskId, assigneeValue)
+                    } catch (_: KeycloakUserNotFoundException) {
```

Comment:
This try catch was made specifically to continue even if there is no user in Keycloak. See [this](https://github.com/valtimo-platform/valtimo-backend-libraries/pull/1489) PR. Please ask @marijnritense @ThomasMinkeRitense